docs/modules/technical-guide/pages/common-concepts/security.adoc

```diff
@@ -329,3 +329,95 @@ The form based login also supports two-factor-authentication.
 
 If the registered `ICredentialVerifier` requests a second factor by returning `AUTH_2FA_REQUIRED`, the `FormBasedAccessController` instructs the UI to show an input field so the user can enter the token.
 To verify this token a second-factor verifier `ICredentialVerifier` needs to be registered.
+
+[[iid-signature]]
+== IId signature
+
+Scout provides a mechanism to add signatures to instances of `IId`. To use this feature simply pass the `IdCodecFlag.SIGNATURE` to the `IdCodec` while serializing or deserializing the id.
+
```

Comment:
Add a sentence about motivation,. e.g. eviate key probing of sequential generated ids to sensitive transactional data